src/main/controllers/features/healthCheckController.ts

```diff
@@ -0,0 +1,5 @@
+import { Request, Response } from "express";
```

Comment:
location of file feature/HealthCheckController.ts @sampankumar Do you think this should here? I am bit confused as well considering structure of the project.
I think if feature/ should be followed by feature name

Reply:
Agreed it can come out of feature folder as well and stays in controller folder directly

Reply:
Also I just noticed that the unit tests are missing